lesson_07/homework_07.py

```diff
@@ -0,0 +1,330 @@
+
+""" Task 08.
+Есть список с числами, написать функцию,
+которая посчитает сумму всех ПАРНЫХ чисел в этом списке.
+Должны браться во внимание только INT значения. Пример взят из ДЗ 6.4.
+"""
+
+print(TASK_START_TEMPLATE.format('08'))
+
+
+def paired_sum(*nums):
+    """
+    Функция, которая суммирует значения парных чисел из списка.
+
+    Функция вычитывает только int значения.
+    """
+    even_int_sum = sum(num for num in nums
+                       if isinstance(num, int) and num % 2 == 0)
+    return f'Сумма парных чисел равна: {even_int_sum}'
+
+
+# Пример использования
+even_sum = paired_sum(10, 12, 1.2, 'qwerty', 13, 22, 15, 'testtest')
+print(even_sum)  # Результат: Сумма парных чисел равна: 44
+
+
+""" Task 09.
+Написать функцию, которая будет считать кол-во букв/слов в тексте.
+Считаются как прописные так и строчные.
+Пользователь предоставляет текст и букву/слово. Текст взят из ДЗ 4.
+"""
+
+print(TASK_START_TEMPLATE.format('09'))
+
+
+def letter_counter(text, letter):
+    """Функция для подсчета выбранной буквы в тексте.
+
+    Функция считает как прописные так и строчные буквы.
+    """
+    let_counter = text.lower().count(letter.lower())
+    return f'В тексте Ваша буква/слово встречается {let_counter} раз.'
+
+
+# Пример использования
+adwentures_of_tom_sawer = """\
+Tom gave up the brush with reluctance in his .... face but alacrity
+in his heart. And while
+the late steamer
+"Big Missouri" worked ....
+and sweated
+in the sun,
+the retired artist sat on a barrel in the ....
+shade close by, dangled his legs,
+munched his apple, and planned the slaughter of more innocents.
+There was no lack of material;
+boys happened along every little while;
+they came to jeer, but .... remained to whitewash. ....
+By the time Ben was fagged out, Tom had
+traded the next chance to Billy Fisher for
+a kite, in good repair;
+and when he played
+out, Johnny Miller bought
+in for a dead rat and a string to swing it with—and so on, and so on,
+hour after hour. And when the middle of the afternoon came, from being a
+poor poverty, stricken boy in the .... morning, Tom was literally
+rolling in wealth."""
+print(letter_counter(adwentures_of_tom_sawer, 'iN'))  # Результат: 15
+
+
+"""Task 10.
+Напишите функцию, которая будет сортировать значения в словаре по
+выбранному параметру. Функция должна выводить значения
+как на увеличение, так и на уменьшение.
+За основу взят словарь из домашнего задания 5 урока.
+"""
+
+print(TASK_START_TEMPLATE.format('10'))
+
+
+def sort_car_data(data):
+    """
+    Функция для сортировки автомобильных данных в списках.
+
+    Функция предлагает выбрать один из 5 вариантов.
+    Функция предоставляет возможность сортировать список
+    по возрастанию и убыванию (пользователь делает выбор).
+    На каждом этапе выбора идет обработка на неправильный ответ.
+    """
+    while True:
+        print('Пожалуйста, выберите параметр сортировки: ')
```

Comment:
Not critical but try to use English language every where 
On real projects - there is no country customization